adafruit_platformdetect/board.py

```diff
@@ -252,6 +254,14 @@ def _pynq_id(self):
         except FileNotFoundError:
             return None
 
+    def _clockwork_pi_id(self):
+        """Check what type of Clockwork Pi board."""
+        board_value = self.detector.get_device_model()
+        board = None
+        if "Clockwork CPI3" in board_value:
```

Comment:
change to `if board_value and "Clockwork CPI3" in board_value:`

Reply:
nice - a tweak will avoid crashes if on a non-clockwork pi

Reply:
Good catch. I haven't programmed python in 15 years so a bit rusty :)   Fixed now!